packages/docs-site/src/components/presenters/Masthead/index.js

```diff
@@ -28,6 +29,14 @@ const MastHead = ({ navItems }) => {
         <a href="/">
           <SiteLogo className="masthead__logo" />
         </a>
+        <div>
+          <a
+            href="https://github.com/Royal-Navy/standards-toolkit"
+            target="blank"
+          >
+            <GitHubLogo className="h_w-6 h_h-6 h_mt-2" />
```

Comment:
Where have these classes come from?

Reply:
These are layout helpers generated by the css-framework.